src/server/v2.0/handler/project.go

```diff
@@ -156,7 +162,7 @@ func (a *projectAPI) CreateProject(ctx context.Context, params operation.CreateP
 	// set the owner as the system admin when the API being called by replication
 	// it's a solution to workaround the restriction of project creation API:
 	// only normal users can create projects
-	if secCtx.IsSolutionUser() {
+	if secCtx.IsSolutionUser() || secCtx.Name() == "robot" {
 		ownerID = 1
```

Comment:
Per discussion with maintainers, we should give the owerID to 0 for robot to avoid any confusion for system. Like in the quota page, the `Owner` column will show the wrong message as robot is not a system admin, given it's a system level robot.

Reply:
does the user id 0 exists in the database?

Reply:
no, it's not in the databsae. Set to 0 is a short term decision. For the long term, we should conside the usage of owner id the Harbor world wide. Whether to reserve it, or sepreate it into two column, reference and reference_type to support both harbor user and robot account.

Reply:
are we save to use a user id, which is not in the database. could we get any problems for example if harbor tries to resolve the owner name by the id?

Reply:
updated the code, now the owner id will be 0 for robot accounts

Reply:
dco check fixed